ci-scripts/README.md

```
@@ -35,6 +45,19 @@ hot-cluster-e2e-run.yml → "Hot Cluster E2E Run"
         └── Playwright gating (or features project)
 ```
 
+**Merge automation (Prow/Tide replacements)**
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)